launch-dedicated-appimage.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+# example launch script, see https://github.com/OpenRA/OpenRA/wiki/Dedicated for details
+
+# Usage:
+#  $ ./launch-dedicated-appimage.sh # Launch a dedicated server with default settings
+#  $ Mod="d2k" ./launch-dedicated-appimage.sh # Launch a dedicated server with default settings but override the Mod
+#  Read the file to see which settings you can override
+
+Name="${Name:-"Dedicated Server"}"
+Mod="${Mod:-"ra"}"
+ListenPort="${ListenPort:-"1234"}"
+AdvertiseOnline="${AdvertiseOnline:-"True"}"
+Password="${Password:-""}"
+
+RequireAuthentication="${RequireAuthentication:-"False"}"
+ProfileIDBlacklist="${ProfileIDBlacklist:-""}"
+ProfileIDWhitelist="${ProfileIDWhitelist:-""}"
+
+EnableSingleplayer="${EnableSingleplayer:-"False"}"
+EnableSyncReports="${EnableSyncReports:-"False"}"
+
+if [ "$Mod" = "d2k" ]; then
+    App="OpenRA-Dune-2000-x86_64.AppImage"
+elif [ "$Mod" = "cnc" ]; then
+    App="OpenRA-Tiberium-Dawn-x86_64.AppImage"
+elif [ "$Mod" = "ra" ]; then
+    App="OpenRA-Red-Alert-x86_64.AppImage"
+else
+    printf "Invalid Mod value %s provided. Allowed values are ra, cnc and d2k" $Mod
```

Comment:
I suppose we want to end the sentence with a fullstop.

Reply:
Updated to include a fullstop